.github/workflows/python.yml

```diff
@@ -21,46 +21,34 @@ jobs:
           - 5432:5432
     steps:
     - name: Setup and run ganache
-      run: |
-        docker run --detach --publish 8545:8545 --network-alias ganache -e DOCKER=true trufflesuite/ganache-cli:latest --defaultBalanceEther 10000 --gasLimit 10000000 -a 30 --noVMErrorsOnRPCResponse --chainId 1337 --networkId 1337 -d
+      run: docker run --detach --publish 8545:8545 --network-alias ganache -e DOCKER=true trufflesuite/ganache-cli:latest --defaultBalanceEther 10000 --gasLimit 10000000 -a 30 --noVMErrorsOnRPCResponse --chainId 1337 --networkId 1337 -d
     - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Create virtualenv
-      run: |
-        python -m venv venv
     - uses: actions/cache@v2
-      id: cache-pip
       with:
-        path: ./venv
-        key: ${{ matrix.python-version }}-${{ hashFiles('requirements-test.txt') }}-${{ hashFiles('requirements.txt') }}
+        path: ~/.cache/pip
```

Comment:
The cached path is different. Is that ok?

Reply:
Yes. The virtual environment is gone so `/venv` no longer exists.

`~/.cache/pip` is the default location on ubuntu (where this action runs)